backend/frontend/templates/frontend/auth/page_login.html

```diff
@@ -0,0 +1,24 @@
+{% extends "frontend/auth/base.html" %}
+<!DOCTYPE html>
+
+{% block content %}
+<div class="mx-auto w-1/3 p-6 bg-white rounded">
+  <h1 class="text-lg">Login</h1>
+  {% if error %}
+  <p>{{ error }}</p>
+  {% endif %}
+
+  <form action="{% url 'user_login' %}" method="post">
+    {% csrf_token %}
+    <label for="username">Username</label>
+    <input class="border" type="text" name="username" id="username" />
```

Comment:
Django has form functionality doesn't it? Why not use that?

Reply:
Oh, yeah, meant to clean this up before marking it ready :)